autokeras/tasks/image.py

```diff
@@ -145,15 +147,15 @@ class ImageRegressor(SupervisedImagePipeline):
     """
 
     def __init__(self,
-                 output_dim=None,
-                 loss='mean_squared_error',
-                 metrics=None,
-                 name='image_regressor',
-                 max_trials=100,
-                 directory=None,
-                 objective='val_loss',
-                 overwrite=True,
-                 seed=None):
+                 output_dim: Optional[int] = None,
+                 loss: str = 'mean_squared_error',
+                 metrics: Optional[List[Union[str, Callable]]] = None,
```

Comment:
Please follow
https://github.com/keras-team/autokeras/blob/7cade0d7853f25d766f35527df14cbd4fdfedb74/autokeras/tasks/text.py#L47
https://github.com/keras-team/autokeras/blob/7cade0d7853f25d766f35527df14cbd4fdfedb74/autokeras/tasks/text.py#L48
to add the type for loss and metrics.  And for the ImageClassifier and ImageSegmenter, too. The original typing for ImageClassifier may not be correct.